encoding/thrift/thriftrw-plugin-yarpc/client.go

```diff
@@ -107,7 +107,12 @@ func (c client) <.Name>(
 	args := <$prefix>Helper.Args(<range .Arguments>_<.Name>, <end>)
 
 	var body <$wire>.Value
+	<if .SanitizeTChannel>
```

Comment:
```
go get -u go.uber.org/thriftrw
go install ./encoding/thrift/thriftrw-plugin-yarpc
thriftrw --plugin=yarpc --out=someTmpDir path/to/thriftFile.thrift
```

This will give you some output to start.